### ISSUE TITLE
Backend broke after installation.

### ISSUE THREAD

**Issue by vicluber** [PIT-Solutions-Private-Limited/ai_translate#8]
(1/1) Error
Call to undefined method TYPO3\CMS\Core\Page\PageRenderer::loadJavaScriptModule()

In typo3 11.5.39 and PHP 8.1.26

**Comment by pitssavinmb** [comment]
Hi vicluber,

The method TYPO3\CMS\Core\Page\PageRenderer::loadJavaScriptModule is only compatible with TYPO3 version 12. Could you please let me know which version of the ai_translate extension you're using?
For TYPO3 version 11, the supported extension is ai_translate_2.1.*.
For TYPO3 version 12, the supported extension is ai_translate_3.1.*.
If you're using the legacy installation mode, please ensure that you have the correct version of the extension installed.

**Comment by vicluber** [comment]
This time is trying to install "pits/ai_translate": "^2.1".

Composer install is throwing this one: Extension "ai_translate" not installed with Composer. This is deprecated and will not work any more with TYPO3 12.
And Typo3 is broken throwing an error because it can't find the repository "DeeplSettingsRepository"
Class "PITS\AiTranslate\Domain\Repository\DeeplSettingsRepository" not found

I'm trying to figure if that's a local problem because the extension was installed correctly last time.
I'll be back.

**Comment by vicluber** [comment]
Finally I manage to install it flawlessly and properly with a normal composer required.
Sometimes is hard to understand what the hell is going on with Typo3. I changed nothing, just checked out an older commit before installing Nitsan t3 ai extension just like I did before but this time the installation went through ok and I can see the backend module from ai_translate.

There are no Languages listed in the backend module tho, but that's a completly different issue. I will close this one now.
